data/links/dslink-javascript-screenshot.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "dslink-javascript-screenshot",
+  "displayName": "Screenshot",
```

Comment:
I think for now anyways, the description should clear up any ambiguity anyone might have, but that's something to keep in mind in the future if we add more ways of taking a screenshot.